mnefun/stats/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Functions for statistical analysis"""
+
+from mnefun import _paths
```

Comment:
use relative paths, not absolute -- and actually this shouldn't be imported in `__init__.py` at all, but rather in whatever function actually needs it